lib/cmds/release.js

```diff
@@ -5,7 +5,7 @@ const command = '*';
 const desc = 'Release a new version (run tests, write changelog, tag version, push release)';
 
 /* istanbul ignore next */
-const handler = () => release();
+const handler = (argv) => release(argv);
```

Comment:
Actually you can call `release` directly. But I guess for consistency it is better to have `handler`.

Reply:
how do you mean that? handler is used in `cli.js`, as we use `commandDir()`